CONTRIBUTOR-LADDER.md

```diff
@@ -0,0 +1,192 @@
+      * Or some equivalent combination or contribution
+   * Must be actively contributing to at least one project area, where the
+   privileges of Organization Member will be beneficial
+
+* Privileges:
+
+   * Can trigger CI
+   * Can leave reviews on PRs
+   * Can recommend other contributors to become Metal3 Github Org Members
+   * Can nominate themselves to [other roles within Metal3](CONTRIBUTOR-ROLS.md)
+
+The process for a Contributor to become an Organization Member is as follows:
+
+1. Open an issue in the [community repo](https://github.com/metal3-io/community)
+1. Assign the issue to a [maintainer](OWNERS).
+
+## Reviewer
+
+Description: A Reviewer has responsibility for specific code, documentation,
+test, or other project areas. They are collectively responsible, with other
+Reviewers, for reviewing all changes to those areas and indicating whether those
+changes are ready to merge. They have a track record of contribution and review
+in the project.
+
+Reviewers are responsible for specific repositories and/or technical areas.
+Most often it is one or more related Git repositories. Reviewers have all the
+rights and responsibilities of an Organization Member.
+
+* Responsibilities:
+
+   * Regularly reviewing Pull Requests against their specific areas of
+    responsibility, including Pull Requests that are assigned to them
+
+   * Helping other contributors become reviewers
+
+* Requirements:
+
+   * Is an Organization Member
+   * Has a track record of constructive and valuable reviews
+   * Has demonstrated knowledge of the specific area, for example by resolving
+   test failures or through contributions
+   * At least 10 Commits in the specific repository.
+   * Is supportive of new and occasional contributors and helps get useful PRs
+   in shape to commit
+
+* Privileges:
+
+   * Has GitHub rights to `lgtm` pull requests in specific directories
+   * Can recommend and review other Org Members to become Reviewers
+
+The process for an Organization Member to become a Reviewer is as follows:
+
+1. Open a PR in specific repository towards the `OWNERS` file and add the
+ persons github handle under `reviewers:` section.
+
+## Approver
+
+An Approver has similar responsibility for specific code, documentation,
+test, or other project areas asa Reviewer. An Approver is however expected to
+have deeper understanding of the code base of the specific repository or technical
+area than a Reviewer and also has shown a substantial track record of
+contribution and review in the project.
+
+Approvers are responsible for specific repositories and/or technical areas.
+Most often it is one or more related Git repositories. Approvers have all the
+rights and responsibilities of a Reviewer.
+
+* Responsibilities:
+
+   * Same as Reviewers
+   * Not only Review the PRs in the specific repositories but also engage/drive
+   in feature proposals and ensure release readiness of the repository.
+
+* Requirements:
+   * Same as Reviewers plus
```

Comment:
```suggestion
   * Same as Reviewers
```
Though I might even to remove it. A higher level always has the same as previous level anyways.